src/core/actions/validators.ts

```diff
@@ -101,10 +101,15 @@ export const isDifferentPageSize: Validator<number> = {
 };
 
 export const isOnDifferentPage: Validator<number> = {
-  func: (page, state) => page !== null && Selectors.page(state) !== page,
+  func: (page, state) => Selectors.page(state) !== page,
   msg: 'page size is already selected'
 };
 
+export const isValidPage: Validator<number> = {
+  func: (page, state) => typeof page === 'number' && Selectors.totalPages(state) >= page && page >= 1,
```

Comment:
I would prefer `page <= Selectors.totalPages(state)` as `page` is the subject of the validation